domains/twab.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "twab-a11y",
+        "email": "twabgaming31@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
it should be records, with "s"